src/components/common-components/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./header/header.component";
```

Comment:
Myślę, że było by fajnie jak by `common-components` były jako modul, czyli żeby importowanie wyglądało jak
`import { Button } from './common-components'` zamiast `import { Button } from './common-components/button-component/button.component'`

Reply:
Zgadzam się, trochę uprościłoby to trochę strukturę, chociaż obecna wersja też nie jest specjalnie problematyczna.